common/alignment/alignment-en.html

```diff
@@ -0,0 +1,82 @@
+---
+{
+	"title": "GCWeb Alignment example",
+	"language": "en",
+	"altLangPage": "alignment-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-19"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2>Horizontal Alignment</h2>
+<h3><code>text-left</code></h3>
+<p class="text-left">Use to align text to the left.</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-left</strong>"&gt;Use to align text to the left.&lt;/p&gt;</code></pre>
+
+<h3><code>text-right</code></h3>
+<p class="text-right">Use to align text to the right.</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-right</strong>"&gt;Use to align text to the right.&lt;/p&gt;</code></pre>
+
+<h3><code>center-block</code></h3>
+<div class="panel panel-default">
+	<div class="panel-body">
+	<img src="https://placehold.it/140x140" class="img-rounded center-block" alt="A generic square placeholder image with rounded corners">
+	</div>
+</div>
+<h4>Code</h4>
+<pre><code>&lt;img <strong>class=&quot;center-block&quot;</strong> alt=&quot;...&quot; src=&quot;x.jpg&quot; /&gt;</code></pre>
+
+<h3><code>text-sm-left</code></h3>
+<p class="text-right text-sm-left">Left-aligned in small view and over</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="text-right <strong>text-sm-left</strong>"&gt;Left-aligned in small view and over&lt;/p&gt;</code></pre>
+
+<h3><code>text-sm-right</code></h3>
+<p class="text-sm-right">Right-aligned in small view and over</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-sm-right</strong>"&gt;Right-aligned in small view and over&lt;/p&gt;</code></pre>
+
+<h2> Vertical alignment </h2>
+<table>
+	<thead>
+		<tr>
+			<th>Class</th>
+			<th>Example</th>
+		</tr>
+	</thead>
+	<tr>
+		<td><code>align-top</code>
+			<p><br /></p></td>
+		<td class="align-top"><p class="margin-top-none margin-bottom-none">Resulting text, top vertical alignment</p></td>
+	</tr>
+	<tr>
+		<td><code>align-middle</code>
+			<p><br /></p></td>
```

Comment:
This should be a table heading as it identify the content of the row.

```suggestion
		<th><code>align-middle</code>
			<p><br /></p></th>
```